services/dy-2Dgraph/use-cases/cc/cc-oned/notebook/cc_oned.ipynb

```diff
@@ -170,8 +170,8 @@
     "    data_frame_a = pd.read_csv(data_path_a, sep=\"\\t\", header=None)\n",
     "    # membrane potential\n",
     "    axis_colums = [0,1]\n",
-    "    plot_data = data_frame_a.filter(items=[data_frame_a.columns[i] for i in axis_colums])\n",
-    "    create_graph(data_frame=plot_data)"
+    "    plot_0 = data_frame_a.filter(items=[data_frame_a.columns[i] for i in axis_colums])\n",
+    "    create_graph(data_frame=plot_0)"
```

Comment:
don;'t see exactly why you 're doing this... but ok...

Reply:
plot_0 is used later on for exporting all the plots to csv